client/checkout/platform-checkout/express-button/use-express-checkout-product-handler.js

```diff
@@ -0,0 +1,113 @@
+/**
```

Comment:
The logic here is mostly copied from [payment-request functions](https://github.com/Automattic/woocommerce-payments/blob/develop/client/payment-request/index.js#L134) but translated from jquery to React.